altapay.php

```diff
@@ -371,23 +371,24 @@ function altapay_meta_box_side( $post_or_order_object ) {
 				if ( $status === 'released' ) {
 					echo '<strong>' . __( 'Payment Released.', 'altapay' ) . '</strong>';
 				} else {
-					$charge = $reserved - $captured - $refunded;
-					if ( $charge <= 0 ) {
-						$charge = 0.00;
-					}
+					$charge        = max( 0, $reserved - $captured - $refunded );
```

Comment:
Are you sure about this?

What if the merchant wants to charge the remaining amount to be captured?

Reserved - 1000 (2 pairs of shoes)
Captured - 500 (1 pair of shoes)
Refunded - 500 (1 pair of shoes) - Did not like model

Charge: 500 (remaining pair of shoes)

Reply:
@emicha I have updated the calculations.